src/mixpanel-core.js

```diff
@@ -67,12 +67,18 @@ var USE_XHR = (window.XMLHttpRequest && 'withCredentials' in new XMLHttpRequest(
 // should only be true for Opera<12
 var ENQUEUE_REQUESTS = !USE_XHR && (userAgent.indexOf('MSIE') === -1) && (userAgent.indexOf('Mozilla') === -1);
 
+var sendBeacon = navigator['sendBeacon'];
+if (sendBeacon) {
+    sendBeacon = _.bind(sendBeacon, navigator);
+}
```

Comment:
I'd personally find the code below more readable if I saw `navigator.sendBeacon`, provides more context even though it's more verbose and you wouldn't need this bind iiuc. Nbd if you prefer it this way.

Reply:
just realized you probably need this to avoid the minifier messing with `'sendBeacon'`; disregard my comment

Reply:
Another size optimization, like we do throughout this SDK. It lets closure compiler compress this down by referring to a minified var rather than injecting "navigator.sendBeacon" every time it's used. The minified version of this code is:
```js
U=I.sendBeacon;U&&(U=c.bind(U,I));
```
so the code below can just refer to `U`